### PR TITLE
Add key-value sample for JavaScript

### LINKED ISSUE #164: Add example using the kv api
Having an example using the kv api would be a great addition.

### PR DESCRIPTION
**Pull request by thorstenhans**
This PR adds an example to illustrate how developers can use key-value APIs using the `spin-js-sdk`

closes #164

### CODE REVIEW

File: examples/javascript/kv/src/index.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
+import { Router, Kv } from '@fermyon/spin-sdk'
+
+const en = new TextEncoder()
+const de = new TextDecoder()
```
Comment (karthik2804) [inline examples/javascript/kv/src/index.js]: Just a tiny nit - could this be renamed as `encoder` and `decoder` to keep it consistent with the other examples?

**Review by karthik2804 — APPROVED** [review]
This is looking great. Thanks!

I just have one small nit.

File: examples/javascript/kv/src/index.js (hunk 1 above)
Comment (thorstenhans) [reply examples/javascript/kv/src/index.js]: Sure will fix it

**Comment by karthik2804** [comment]
Thank you once again for tackling this @ThorstenHans!

**Pull request merged**